client/src/partials/cash/modals/invoices.modal.js

```diff
@@ -88,13 +88,16 @@ function CashInvoiceModalController(Debtors, debtorId, invoiceIds, ModalInstance
     vm.loadingState = !vm.loadingState;
   }
 
-
   // resolve the modal with the selected invoices to add to the cash payment bills
   function submit() {
 
     // retrieve the outstanding patient invoices from the ui grid
     var invoices = vm.getSelectedRows();
 
+    // block the submission if there are no
```

Comment:
Perhaps - however, the above code will never set `vm.empty` to false.  I might want to use it for further validation in the future :).